app/javascript/packages/address-search/components/in-person-location-redirect-alert.tsx

```diff
@@ -0,0 +1,26 @@
+import { Alert } from '@18f/identity-components';
+import { t } from '@18f/identity-i18n';
+import { getConfigValue } from '@18f/identity-config';
+
+interface InPersonLocationRedirectAlertProps {
+  infoAlertURL?: string;
+}
+
+function InPersonLocationRedirectAlert({
+  infoAlertURL = 'https://login.gov/',
+}: InPersonLocationRedirectAlertProps) {
+  const infoAlertText = t('in_person_proofing.body.location.po_search.you_must_start.message', {
+    app_name: getConfigValue('appName'),
```

Comment:
It could probably work in the brochure site as long as the associated configuration "script" is provided in the template.

i.e.

```html
<script type="application/json" data-config>{"appName":"Login.gov"}</script>
```

Reply:
I could also make this a value passed to the component, if that'd make things easier on the brochure site.

Reply:
changed the approach in 331753c1672554f80d929798cd2fb8a81ad45ca9